redditfunction.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+import requests
+import json
+
+class RedditFunction :
+    
+    #This function returns a list of reddit post links
+    def getPushshiftData(self, size, after, before, queryr) :
+        url = 'https://api.pushshift.io/reddit/search/submission/?&sort=desc&sort_type=score&over_18=false&score=>2000&size=' +str(size) + '&after=' + str(after) + '&before=' +str(before) + '&q=' + str(queryr)

```

Comment:
A little cleaner and more descriptive way of building GET parameters: https://stackoverflow.com/a/44552191

Reply:
Thank you, I may refactor down the line, but for now the URLs will all be consistent so it shouldn't need to check for slashes, but I do like the cleaner arguments layout so I will bookmark this for the future.